pennylane/ops/common.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
Hi @dime10, thank you for these changes! :slightly_smiling_face: 

I'm wondering if we could simply just name this file `identity.py`. It would describe its content well. There aren't any further ops that will be shared between the qubit and CV models and we could rename the file later if required.

Reply:
Sure we can do that instead :)